test/libsolidity/gasTests/abiv2_optimised.sol

```diff
@@ -17,9 +17,9 @@ contract C {
 // optimize-yul: true
 // ----
 // creation:
-//   codeDepositCost: 614600
-//   executionCost: 651
-//   totalCost: 615251
+//   codeDepositCost: 597000
+//   executionCost: 632
+//   totalCost: 597632
```

Comment:
A [diff](https://gist.github.com/hrkrshnn/d6f1178ec2fcc21f3b40cd2af32edbfc#file-diff-diff) of the changes. Some functions with unused function parameters exist, so we can further decrease the costs. But still the gas costs are lower than [this point](https://github.com/ethereum/solidity/pull/9397#pullrequestreview-449793452)